bundles/org.openhab.binding.flicbutton/src/3rdparty/LICENSE

```diff
@@ -0,0 +1,121 @@
+Creative Commons Legal Code
```

Comment:
Can't this library be added via a dependency?

Reply:
Unfortunately, no. Shortcut Labs does not provide a built version, so we have to maintain the build of it.